packages/react-server/core/logging/server.js

```diff
@@ -68,7 +68,7 @@ function errorInterceptor (level, msg, meta) {
 // massage the error into a format suitable for logging
 function normalizeError (err) {
 	if (err instanceof Error) {
-		return _.pick({
+		return _.filter({
```

Comment:
Nice catch @roblg.  Definitely want to keep this an object.

Looks like this broke during [the lodash upgrade](https://github.com/redfin/react-server/pull/137).  `_.pick` used to take a key array _or_ a predicate, but the predicate variant [got split out into `_.pickBy` in v4.0.0](https://github.com/lodash/lodash/wiki/Changelog#v400):

> Split _.omit & _.pick into _.omitBy & _.pickBy

Will update to use `_.pickBy`.  Thanks!